.github/workflows/build.yml

```diff
@@ -20,12 +15,18 @@ jobs:
           fetch-depth: 0
 
       - name: Homebrew install dependencies
-        # Compared to the README, adds ccache for faster compilation times and removes sh5sum to prevent some conflict with coreutils
```

Comment:
sh5sum seems to have been removed in #5891

Reply:
sh5sum is removed, see line 21 to 23

Reply:
Yes, but you changed the comment without saying why it was justified to do so in the commit message or PR. ;-)

Reply:
This is for me to put in the squashed commit message, in other words.

Reply:
I just follow the build guide, it does not have sh5sum https://github.com/koreader/koreader/blob/master/doc/Building.md#macos

Reply:
And for future reference, that should be stated in the commit message. ;-) But I already said it was a remark from me to me. ^_^